examples/api/lib/material/paginated_data_table/paginated_data_table.1.dart

```diff
@@ -62,12 +62,23 @@ class DataTableExampleApp extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return const MaterialApp(
+    return MaterialApp(
+      scrollBehavior: MyMaterialScrollBehavior(),
       home: SingleChildScrollView(padding: EdgeInsets.all(12.0), child: DataTableExample()),
     );
   }
 }
 
+class MyMaterialScrollBehavior extends MaterialScrollBehavior {
```

Comment:
Super nit: Would it be better to make this private so readers know it's not part of the framework?